dt-assets/js/shared-functions.js

```diff
@@ -187,9 +192,9 @@ function handleAjaxError(err) {
 
 jQuery(document)
   .ajaxComplete((event, xhr, settings) => {
-    if (xhr && xhr.responseJSON) {
+    if ( xhr && xhr.responseJSON && settings.type === "POST" ) {
```

Comment:
These changes are already in the theme

Reply:
@kodinkat this one referred to the whole file.

Reply:
@corsacca yep, my bad! Just pushed clean-up! :)